src/main/java/de/danielbechler/diff/selector/CollectionItemElementSelector.java

```diff
@@ -71,6 +94,6 @@ public boolean equals(final Object o)
 	@Override
 	public int hashCode()
 	{
-		return item != null ? item.hashCode() : 0;
+		return item != null ? identityStrategy.hashCode(item) : 0;
```

Comment:
Ok, but I don't fully understand what the problem is with hashCode. It is reliable if in the `IdentityStrategy` it is implemented as per normal with its equals pair.
With other words, I don't understand why the constant is better.